pkg/version/remote.go

```diff
@@ -28,6 +30,10 @@ import (
 	"go.thethings.network/lorawan-stack/v3/pkg/log"
 )
 
+const (
+	versionCheckCacheTTL = 11 * time.Hour // cache is valid for one hour less than component.version_check.versionCheckPeriod
```

Comment:
Comments are sentences; they start with an uppercase letter and end with a period. More below.